### ISSUE TITLE
Accessibility - Announcement does not work if searchable is true

### ISSUE THREAD

**Issue by yrebsamen** [ng-select/ng-select#2589]
### Describe the bug

For our website, I want to use an ng-Select that allows you to search for a country or select a country via a dropdown. The problem is that the input field is only available with [searchable]="true." However, when I use this and navigate with the tab key, no announcement appears. However, if I set it to false, it is announced correctly.

### Steps to Reproduce

Ng-Select with [searchable]="true" and a screen reader.

### Please provide a link to a minimal reproduction of the bug. StackBlitz, CodePen or CodeSanBox

-

### Expected behaviour

the ng select should be announced properly.

### What version of the library you see this issue?

14.1.0

### What version of Angular are you using?

Angular 19
